# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/palo/force-64bit.patch.hppa64
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- palo-2.24/ipl/ipl.c.vanilla	2024-04-10 17:56:22.913957057 +0200
+++ palo-2.24/ipl/ipl.c	2024-04-10 17:58:25.968956517 +0200
@@ -791,17 +791,20 @@
 		    goto restart;
 		}
 	    }
-
+#if 0
 	    if (kernfd == -1 && (pdc_os_bits() & PDC_MODEL_OS64))
 	    {
+#endif
 		printf("Choosing 64-bit kernel\n");
 		kernfd = offset_open(bootdev, f.kern64_offset, f.kern64_sz);
+#if 0
 	    }
 	    else if (kernfd == -1 && (pdc_os_bits() & PDC_MODEL_OS32))
 	    {
 		printf("Choosing 32-bit kernel\n");
 		kernfd = offset_open(bootdev, f.kern32_offset, f.kern32_sz);
 	    }
+#endif
 	}
 
 	if (!load_kernel(kernfd, &entry, &wide))
